pulse_actions/publisher.py

```diff
@@ -16,10 +16,10 @@
 
 
 class ExperimentalPublisher(GenericPublisher):
```

Comment:
We should probably rename this to PulseActions publisher if it is no longer just an experiment.